xbmc/utils/log.h

```diff
@@ -33,6 +33,7 @@
 #include <string>
 #include <vector>
 
+#include <fmt/chrono.h>
```

Comment:
Why this change?

Reply:
Was required to get it compiled on Linux x86_64.

But all Windows builds are failing in jenkins with libfmt errors. Therefore I will revert this PR to my minimal fix in a few hours.

Reply:
Problem is most probably the call to CLog::Log with delta now having type ‚milliseconds‘ instead of int

Reply:
Try putting the `#include <fmt/chrono.h>` in  RepositoryUpdater.cpp, that should make Windows happy.

Reply:
Not only Windows, also macOS (just tested).

Reply:
@ksooo Windows errors are in UPNP modules.

@neo1973 Just pushed the untested change. May Jenkins decide.